engine/src/main/java/org/terasology/config/AudioConfig.java

```diff
@@ -31,39 +31,63 @@
 
     private transient PropertyChangeSupport propertyChangeSupport = new PropertyChangeSupport(this);
 
+    /**
+     * @return Returns a float of the sound volume
+     */
     public float getSoundVolume() {
         return soundVolume;
     }
 
+    /**
+     * @param soundVolume Sets the sound volume starting from the previous value and going to the soundVolume passed in
+     */
     public void setSoundVolume(float soundVolume) {
         float oldValue = this.soundVolume;
         this.soundVolume = soundVolume;
         propertyChangeSupport.firePropertyChange(SOUND_VOLUME, oldValue, soundVolume);
     }
 
+    /**
+     * @return Returns music volume
+     */
     public float getMusicVolume() {
         return musicVolume;
     }
 
+    /**
+     * @param musicVolume Sets music volume using the same principle as setSoundVolume()
```

Comment:
I'd recommend to cut, paste and adapt whatever you write in the javadoc for setSoundVolume(). Remember that somebody might be reading this particular javadoc via an html page and might be jumping to this method right from a link, skipping setSoundVolume() altogether: we don't want that person to have to backtrack to it.